### ISSUE TITLE
error saying SpecialFeatureOrQuotaIdRequired 

### ISSUE THREAD

**Issue by maythamfahmi** [Azure-Samples/azure-devops-copilot-extension#19]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> After the latest fixing of issue #10, when I tried to deploy it from the portal, and got a new error:

![image](https://github.com/Azure-Samples/azure-devops-copilot-extension/assets/9260645/71594249-bc6b-4ad1-b393-89b4c0aa98da)

error text
```
{"code":"InvalidTemplateDeployment","details":[{"code":"SpecialFeatureOrQuotaIdRequired","message":"The subscription does not have QuotaId/Feature required by SKU 'S0' from kind 'OpenAI' or contains blocked QuotaId/Feature."}],"message":"The template deployment 'Microsoft.Template-20240326231843' is not valid according to the validation procedure. The tracking id is '5bcc38e8-2a11-43b7-87e4-40075ed15497'. See inner errors for details."} 
```

The same issue is from Azure CLI, this happened when I chose a location West Europe

`az deployment sub create --location westeurope --template-file main.bicep`

error
```
{"code": "InvalidTemplateDeployment", "message": "The template deployment 'main' is not valid according to the validation procedure. The tracking id is 'ee1b972a-1e2f-476a-b24a-0436698c9445'. See inner errors for details."}

Inner Errors:
{"code": "SpecialFeatureOrQuotaIdRequired", "message": "The subscription does not have QuotaId/Feature required by SKU 'S0' from kind 'OpenAI' or contains blocked QuotaId/Feature."}
```

When I the other hand chose the default eastus or Sweden Central, it gives another error 

`az deployment sub create --location eastus --template-file main.bicep`

error
```
{"code": "InvalidDeploymentLocation", "message": "Invalid deployment location 'swedencentral'. The deployment 'MAIN' already exists in location 'westeurope'."}
```

### Any log messages given by the failure

>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by mahomedalid** [comment]
Is your subscription OpenAI enabled?

You may need to submit a request for access here: https://customervoice.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7en2Ais5pxKtso_Pz4b1_xUNTZBNzRKNlVQSFhZMU9aV09EVzYxWFdORCQlQCN0PWcu

**Comment by maythamfahmi** [comment]
That is correct, I have applied and got approved, but now getting another error, I tried changing the name to `devops-copilot-0001`

Invalid deployment location 'eastus'. The deployment 'DEVOPS-COPILOT-RG' already exists in location 'swedencentral'. (Code: InvalidDeploymentLocation)

![image](https://github.com/Azure-Samples/azure-devops-copilot-extension/assets/9260645/b875c3a9-8b9a-42dc-b022-fd4d2330e756)

And I tried from the command line, it says

`az deployment sub create --location swedencentral --template-file main.bicep`

{"code": "InvalidDeploymentLocation", "message": "Invalid deployment location 'swedencentral'. The deployment 'MAIN' already exists in location 'westeurope'."}

**Comment by mahomedalid** [comment]
Thanks for reporting, I should be able to fix this.

In the meanwhile, delete existing resource groups related to this project, and go to your subscription > deployments and delte de azure devops copilot deployment as well.

**Comment by maythamfahmi** [comment]
@mahomedalid, Do you have any updates about the above? ☝️
